libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -507,6 +558,31 @@ bool AP_Mission::read_cmd_from_storage(uint16_t index, Mission_Command& cmd) con
     return true;
 }
 
+bool AP_Mission::stored_in_location(uint16_t id)
```

Comment:
This is not a win IMO. It makes adding new commands harder, as you have to remember to check this block as well, I'd prefer to retain the copy_location flag.

Reply:
Previously we had two places which encoded this information.

storage/retrieval would have added another two.  That's four places to remember to change....

IF your new command only uses location then there's only one place to modify.  That's a win.